data/core/common.lua

```diff
@@ -110,6 +170,16 @@ local function fuzzy_match_items(items, needle, files)
 end
 
 
+---Performs fuzzy matching.
+---
+---If the haystack is a string, a score ranging from 0 to 1 is returned. </br>
```

Comment:
Is the br tag something sumneko expects?

Reply:
Yes. It's to add a newline without adding a linebreak. The other ways of doing it is to add `\n`, but I think </br> is clearer here.

It's just a compromise I made to make the description actually readable within LSP. I believe no one would have a problem with having that there.